corehq/apps/users/models.py

```diff
@@ -112,6 +112,8 @@
 MAX_WEB_USER_LOGIN_ATTEMPTS = 5
 MAX_COMMCARE_USER_LOGIN_ATTEMPTS = 500
 
+CURRENT_VERSION = '3.0'  # Set this to the most up to date version of the eula
```

Comment:
Since this is now visible at the top level of this module, I'd name it something more explicit, like `EULA_CURRENT VERSION`.

Another option is to not pull out the constant here, but reference it in the view with `EulaMixin.CURRENT_VERSION`.

Reply:
I think in makes sense to go with keeping `CURRENT_VERSION` where it is and referencing  `EulaMixin.CURRENT_VERSION` in the view. Things could be rearranged one day when/if a constants file gets made.